internal/cmd/config/profile/set/set.go

```diff
@@ -67,9 +68,21 @@ func NewCmd(p *print.Printer) *cobra.Command {
 }
 
 func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
-	profile := inputArgs[0]
+	var err error
+	var profile string
+
+	// Read profile name
+	if len(inputArgs) > 0 {
+		profile = inputArgs[0]
+	} else {
+		var profileSet bool
+		profile, profileSet = config.GetProfileFromEnv()
+		if !profileSet {
+			return nil, &errors.ProfileNameNotProvided{}
+		}
+	}
```

Comment:
What is the use case of setting a profile using an env var?

Using the env var or any command (e.g. `STACKIT_CLI_PROFILE=alt-profile stackit dns zone list`) will execute that command using the `alt-profile`, overriding any profile that might be set in the file, right?

Reply:
Good point, its not very good functionality. Removed